Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1382,6 +1460,28 @@ def update_remote_system_command(args, params, service: client.Service, auth_tok
     else:
         demisto.debug(f'Incident corresponding to notable {notable_id} was not changed.')
 
+    if entries:
+        for entry in entries:
+            entry_tags = entry.get('tags', [])
+            demisto.debug(f'Got the entry tags: {entry_tags}')
+            if comment_tag_to_splunk in entry_tags:
+                demisto.debug('Add new comment')
+                comment_body = f'{entry.get("contents", "")}\n {COMMENT_MIRRORED_FROM_XSOAR}'
+                demisto.debug(f"will update comment: {comment_body}")
+                base_url = 'https://' + params['host'] + ':' + params['port'] + '/'
+                try:
+                    session_key = get_auth_session_key(service) if not auth_token else None
+                    response_info = update_notable_events(
+                        baseurl=base_url, comment=comment_body, auth_token=auth_token, sessionKey=session_key,
+                        eventIDs=[notable_id])
+                    if 'success' not in response_info or not response_info['success']:
+                        demisto.error(f'Failed updating notable {notable_id}: {str(response_info)}')
+                    else:
+                        demisto.debug('update-remote-system for notable {}: {}'
+                                      .format(notable_id, response_info.get('message')))
+                except Exception as e:
+                    demisto.error('Error in Splunk outgoing mirror for incident corresponding to notable {}. '
```

Comment:
not 100% sure, but it looks like this should have been a method like "update entries" or something like that. and also the building of a base url seems displaced. this should be already handled by the client.